src/main/java/com/laytonsmith/core/functions/PlayerManagement.java

```diff
@@ -2090,17 +2092,21 @@ public Construct exec(Target t, Environment env, Construct... args) throws Confi
 			int strength = Static.getInt32(args[2], t);
 			int seconds = 30;
 			boolean ambient = false;
+			boolean particles = true;
 			if (args.length >= 4) {
 				seconds = Static.getInt32(args[3], t);
 			}
 			if (args.length == 5) {
```

Comment:
This needs to be changed, as you still want to read in args[4] if the length is either 5 OR 6. Change it to args.length >= 5. Also, use args.length >= 6 below, just for good measure.